.github/workflows/ci.yml

```
@@ -48,6 +48,11 @@ jobs:
           sudo apt install llvm-15 # for opt-15, used to simplify LLVM for reference semantics.
           cd test/bruteforce-correctness && ./run.sh
 
+      - name: 'Check for changes in AliveStatements.lean'
+        run: |
+          (cd SSA/Projects/InstCombine/; update_alive_statements.py)
+	  git diff | grep .
```

@tobiasgrosser Might I suggest 

```
set -e expandtab
set tabstop=4
```

in your vimrc?

```suggestion
          (cd SSA/Projects/InstCombine/; update_alive_statements.py)
          git diff | grep .
```

---

Done.